pkgs/tools/security/ecryptfs/default.nix

```diff
@@ -56,7 +56,7 @@ stdenv.mkDerivation rec {
     done
   '';
 
-  configureFlags = lib.optionals (!enablePython) [ "--disable-pywrap" ];
+  configureFlags = lib.optionals (!enablePython) [ "--disable-pywrap --disable-openssl" ];
```

Comment:
This will still pull in openssl when `enablePython`?

Reply:
oops, true, fixed